modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -760,4 +760,25 @@ definition {
 			locator1 = "PRMMDFRequest#STATUS_BUTTON");
 	}
 
+	@summary = "Assert the value for Budget MDF Request"
+	macro viewValueBudget{
+		if (isSet(totalCost)) {
+			AssertTextEquals(
+				key_fieldName = "Total cost",
+				key_fieldValue = ${fieldValue},
+				locator1 = "PRMMDFRequest#VALUE_BUDGET");
+		}
+
+		if (isSet(claimPercent)) {
+			AssertTextEquals(
+				key_fieldName = "Claim Percent",
+				key_fieldValue = ${fieldValue},
+				locator1 = "PRMMDFRequest#VALUE_BUDGET");
+		}
+	}
+
+	@summary = "Verify that the budget value will be cleaned"
```

Comment:
Hi @Italo-gabriel, your summary says 'Verify that the budget value will be cleaned', but you aren't verifying anything. You are clicking in a button, so your summary should match with what your macro is doing.
So it can be 'Click on the '-' button, inside the activities page of the MDF Request, to remove a budget value'. Does that make sense to you?